packages/cli/examples/README.md

```
@@ -0,0 +1,306 @@
+walkeros bundle examples/server-collect.json
+walkeros run collect examples/server-collect.mjs -p 8080
+```
+
+**Terminal 2 - Start web server**:
+
+```bash
+walkeros bundle examples/web-serve.json
+walkeros run serve examples/web-serve.mjs -p 3000
+```
+
+**Browser**: Open http://localhost:3000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)